shotgun_api3/shotgun.py

```diff
@@ -66,19 +66,20 @@
 
 SG_TIMEZONE = SgTimezone()
 
-
 NO_SSL_VALIDATION = False
 try:
-    import ssl
-    if os.environ.get("SHOTGUN_DISABLE_SSL_VALIDATION", False):
+    import ssl        
+except ImportError, e:
+    if os.environ.get("SHOTGUN_FORCE_CERTIFICATE_VALIDATION") not in [0, "0", "", None]:
```

Comment:
I agree. Was responding to a pass comment from Manne about this being a bit too uncertain as so what the acceptable value(s) are as well as differences bewteen how platforms handle them. But in this case, the value doesn't matter... so I will happily change this :)